### ISSUE TITLE
Gif not working?

### ISSUE THREAD

**Issue by sprueshoes** [GyanD/codexffmpeg#143]
Hi, total newbie here. I downloaded this to use with Opentoonz but the gif render feature doesn't work. I'm seeing another user posting [here](https://www.reddit.com/r/OpenToonz/comments/1cma67k/rendering_gif_not_work/) that it is a known problem. Is this the right place to post this issue? Thanks a ton!

**Comment by GyanD** [comment]
'doesn't work' is vague. What happens? Which version of ffmpeg? Which version of Opentoonz?

**Comment by sprueshoes** [comment]
Opentoonz 1.7.1. I was able to select gif as a file type for rendering. It runs the dialogue that says "Rendering frames etc" and when that dialogue is finished it closes and pops a new dialogue box that says something along the lines of "Finalizing rendering" but it closes very quickly. It fails to produce anything in the output folder. Other (non ffmpeg) rendering outputs using other file types work fine. I used the most recent build as of 6/19/24. Thank you so much for looking at this.
